src/pages/index.js

```diff
@@ -173,14 +168,16 @@ function handleAddNewCardFormSubmit(inputValues) {
       //close popup after submit
       newCardPopup.closeModal();
     })
+    .catch((err) => {
+      console.log(err);
+    })
     .finally(() => {
-      document.querySelector("#create-button").textContent = "Create";
+      constant.createButton.textContent = "Create...";
```

Comment:
* See here http://joxi.ru/krDQJBDhdOYYqm the button text becomes `Create...` after the first submit. But it should be just `Create` without any dotes


`      createButton.textContent = "Create";
`